sdk/appconfiguration/app-configuration/package.json

```diff
@@ -89,7 +89,7 @@
     "@azure/core-lro": "^2.5.1",
     "@azure/core-paging": "^1.4.0",
     "@azure/core-rest-pipeline": "^1.6.0",
-    "@azure/core-tracing": "^1.0.0",
+    "@azure/core-tracing": "1.1.3",
```

Comment:
pin to 1.1.3, while test-utils pins to 1.1.2 - this allows me to update test-utils without patching node_modules